org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/resource/IStorage2UriMapperJdtExtensions.java

```diff
@@ -26,10 +28,28 @@
 	/**
 	 * @return all managed uris and their corresponding storages contained in the given {@link IPackageFragmentRoot}
 	 */
-	public Map<URI, IStorage> getAllEntries(IPackageFragmentRoot root);
+	Map<URI, IStorage> getAllEntries(IPackageFragmentRoot root);
```

Comment:
change intentional?

Reply:
public is the default in interfaces. Yes, intentational.